.github/workflows/docs.yml

```
@@ -15,13 +15,8 @@ concurrency:
 
 jobs:
   documentation:
-
-    name: Build docs on ${{ matrix.os }}
-    runs-on: ${{ matrix.os }}
-
-    strategy:
-      matrix:
-        os: [ubuntu-latest]
+    name: Build docs on ubuntu-latest
```

Might be good enough to optionally make it even shorter.

```suggestion
    name: Build docs
```